### PR TITLE
2048: Tweaks before production

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

This PR tries todo what is required to get 2048 online.

  - [x] Added `2048` trigger
  - [x] Fix broken alignment issue with UI
  - [x] Fix IA tab bug on mobile

## Related Issues and Discussions

https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2825

## People to notify

@moollaza 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [x] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [x] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/game2048

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by pjhampton** [comment]
Currently, there is an issue with the tabs that requires us to put this issue on hold again or support mobile touch events. In screenshot 1, the `Answer` tab is displayed, but when clicked on, it disappears.

**Screenshot 1**
img_1789

**Screenshot 2**
img_1790

**Comment by moollaza** [comment]
@pjhampton thanks! We'll need to find a better way of allowing the Goodie to not trigger on mobile devices before we can launch this. I'll be investigating that shortly.

**Comment by moollaza** [comment]
@pjhampton do you want to investigate adding swipe events?

Seems like this is a simple vanilla JS approach: http://stackoverflow.com/a/23230280/819937

Alternatively we might be able to use something small like http://hammerjs.github.io/

**Comment by pjhampton** [comment]
Yep, will send a PR after the calculator P2 😄

**Comment by pjhampton** [comment]
Ping @bsstoner

**Comment by pjhampton** [comment]
I believe I have resolved this 👍

File: share/goodie/game2048/game2048.js [hunk 1]
```diff
@@ -313,15 +313,22 @@ DDH.game2048.build = function(ops) {
...
-    return {
-        onShow: function() {
-            //Hide this goodie on mobile devices for now
-            if(is_mobile || is_mobile_device) {
-                DDH.spice_tabs.game2048.hideLink();
-                DDH.spice_tabs.game2048.hide();
-                return;
-            }
+    
+    DDH.game2048.build_async = function(ops, DDH_async_add) {
```
Comment (moollaza) [inline share/goodie/game2048/game2048.js]: With the recent changes I made, you don't need to use async -- you just need to return false before the build function returns 👍 

I'll commit a simplified approach

**Comment by moollaza** [comment]
@pjhampton Thanks! This LGTM :+1:

**Pull request merged**